site/content/en/docs/api_sdk/cli/_index.md

```diff
@@ -316,3 +321,25 @@ see that command's examples for more information.
   ```bash
   cvat-cli project ls --json > list_of_projects.json
   ```
+
```

Comment:
Not related to the PR, but it would be nice to extend support for showing subcommand help without requiring to authenticate in CVAT. Currently, doing something like this: `cvat-cli --auth "user" function run-agent -h` results in a password being asked.

Reply:
Yeah, I agree. I really is unrelated, though. 🙂